modules/cli/src/main/scala/scala/cli/commands/Run.scala

```diff
@@ -103,11 +103,36 @@ object Run extends ScalaCommand[RunOptions] with BuildCommandHelpers {
   }
 
   def run(
-    options: RunOptions,
+    opts: RunOptions,
     inputArgs: Seq[String],
     programArgs: Seq[String],
     defaultInputs: () => Option[Inputs]
   ): Unit = {
+    val options =
+      if (!opts.sharedRun.sharedDebug.debug)
```

Comment:
IMO we can make it slightly smarter.
if the user passes `--debug-port 5005` or `--debug-mode listen`, but he didn't pass the `--debug` flag, we probably can assume he still intended to run in debug mode.

so I think the following syntax variants should all be accepted (and any permutations thereof):
```
scala-cli run . --debug
scala-cli run . --debug-port 5005
scala-cli run . --debug-mode listen
```